docs/source/conf.py

```diff
@@ -0,0 +1,54 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'PyDAX'
+copyright = '2020, IBM'
+author = 'Edward Leardi, Hong Xu'
```

Comment:
I'm wondering what our tradition is about authors. Do we just say "IBM" or actually list out people?

Reply:
One thing we can do is to actually having this field vaguely (because in the future there is likely some contribution from outside). Some files like this: https://github.com/Python-Markdown/markdown/blob/master/docs/authors.md, or this: https://github.com/sphinx-doc/sphinx/blob/3.x/AUTHORS . In other words, we can have a more dedicated doc page on who these people are, and remove the field here and simply emphasize that IBM is the copyright owner.

Reply:
ok that sounds well thought out, we can go with that